testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -27,9 +27,11 @@ class TestLeafletFinder(TestCase):
     def setUp(self):
         self.universe = MDAnalysis.Universe(Martini_membrane_gro, Martini_membrane_gro)
         self.lipid_heads = self.universe.select_atoms("name PO4")
+        self.lipid_head_string = "name PO4"
 
     def tearDown(self):
         del self.universe
+        del self.lipid_head_string
```

Comment:
I like to be explicit about the unit test clean up process. Seems like a matter of personal preference.